src/System.Data.SqlClient/src/System/Data/SqlClient/SqlParameterCollection.cs

```diff
@@ -30,6 +30,21 @@ internal bool IsDirty
                 _isDirty = value;
             }
         }
+        override public bool IsFixedSize
```

Comment:
Nit: 
`public override` instead of `override public`
`public override bool IsFixedSize => ((System.Collections.IList)InnerList).IsFixedSize;`

Reply:
Updated in the new commit.